src/Opserver.Web/Opserver.Web.csproj

```diff
@@ -9,7 +9,7 @@
     <PackageReference Include="BuildBundlerMinifier" Version="3.2.449" PrivateAssets="all" />
     <PackageReference Include="BuildWebCompiler" Condition="'$(OS)' == 'Windows_NT'" Version="1.12.405" PrivateAssets="all" />
     <PackageReference Include="MiniProfiler.AspNetCore.Mvc" Version="4.2.22" />
-    <PackageReference Include="Microsoft.IdentityModel.Protocols.OpenIdConnect" Version="6.15.0" />
+    <PackageReference Include="Microsoft.IdentityModel.Protocols.OpenIdConnect" Version="6.34.0" />
```

Comment:
Maybe we could submit a PR upstream instead of to our own repo?

Reply:
Lets do that.

BTW. How did you arrive in this PR? Just curious. The slack-automaton hasn't kicked in yet. 👁️

Reply:
I have a [bookmark](https://github.com/pulls?q=is%3Apr+is%3Aopen+label%3A%22Cell%3ATwilight+Sparkle%22+label%3A%22ready+for+review%22) to our PRs with `read-for-review` :)

Reply:
> I have a [bookmark](https://github.com/pulls?q=is%3Apr+is%3Aopen+label%3A%22Cell%3ATwilight+Sparkle%22+label%3A%22ready+for+review%22) to our PRs with `read-for-review` :)

Nice! I've booked marked too.